src/gossip_agent/mod.rs

```diff
@@ -93,6 +93,7 @@ impl GossipAgent {
         subtopics: Option<Vec<&str>>,
         waku_host: Option<String>,
         waku_port: Option<String>,
+        waku_addr: Option<String>,
```

Comment:
I think we should rename these three, the end user shouldn't have to care what Waku is, we can call them `graphcast_host`, `graphcast_port`, `node_addr`/`boot_node_addr`, what do you think?

Reply:
I wouldn't label them with graphcast, and I think that since they are in the SDK level then those who are developing with the SDK should be aware that waku is the underlying gossip protocol and that they will be communicating through waku nodes?

Reply:
I still don't think they should be aware that we are using Waku, it's unnecessary cognitive load for the SDK items, especially user-facing ones.  Graphcast should be a black box in that aspect, and to the users it shouldn't matter what underlying gossip protocol we use. If we were to change the protocol for instance, we would want the field names to stay the same